src/app/public/public_api.ts

```diff
@@ -1,10 +1,11 @@
-export * from './config-module-for-root-args';
+export * from './app-config-host';
+export * from './app-config-params';
+export * from './app-config-module-for-root-args';
+export * from './app-config.module';
```

Comment:
Renamed the files, too, to make it easier to grasp the naming conventions (for future developers).